magic_bridge/ic/src/magic_bridge/src/magic.rs

```diff
@@ -174,6 +210,7 @@ impl MagicState {
         StableMagicState {
             canisters: self.canisters.take(),
             controllers: self.controllers.take(),
+            dip20_reference: self.dip20_reference.take(),
```

Comment:
whats `dip20_reference`?

Reply:
I know its not the best naming... but the idea is to have <eth_address_as_principal, status> and the status is not just the canister status it also includes some states such as "NotCreated" which means magic have been called and tried to create the dip20 but failed on the first step.